README.md

```diff
@@ -54,52 +59,54 @@ Just stick this in your `<head>`:
 
 #### Enforce a theme and ignore `(prefers-color-scheme)`
 
-For the main versions, `dark` or `light` is only treated as a _default theme_: if a user has a system-wide preference for either dark or light mode on their device, `water.css` will respect this. If you want to avoid this behavior and enforce dark or light theme, append `.standalone` to the theme prefix, e.g. `dark.standalone.min.css`.
+For the main `water.css` file, dark is only treated as a _default theme_: if a user has a preference for either dark or light mode on their device, the stylesheet will respect this. This detection is made possible through a recent CSS media query called `prefers-color-scheme`. If you want to avoid this behavior and enforce dark or light theme, use either `dark.css` or `light.css`.
 
 #### Want to support Internet Explorer?
 
-Sure, just extend the theme prefix with `-legacy`, e.g. `dark-legacy.min.css`.
-Be aware that these versions **do not support** [runtime theming](#theming) as they use hard coded values rather than variables. Additionally, if you use a legacy version that is not standalone, we recommend [you add the respective preload tags to improve load times](https://watercss.netlify.com/?legacy#installation).
+Both `dark.css` and `light.css` support Internet Explorer 11, but the main `water.css` file **does not** due to lack of `prefers-color-scheme` support.
```

Comment:
I think we should mention here that the main water.css *does* support IE, it just won't switch between dark and light automatically. (right?)
With the current wording it sounds like choosing water.css means that users on IE will see broken styles.

Reply:
Oh yeah, good point. I'll fix that, thanks!

Reply:
@jonaskuske I updated that section to the following, does it make more sense?

All three distributions of Water.css support Internet Explorer 11, but the main `water.css` file **doesn't respect the user's color scheme** and will be locked to dark mode due to lack of `prefers-color-scheme` support.

Reply:
LGTM 👍